ui/packages/consul-ui/app/models/node.js

```diff
@@ -45,16 +47,16 @@ export default class Node extends Model {
 
   @computed('Checks.[]')
```

Comment:
Worth updating these dependency lists to `NodeChecks.[]`?

Also: I’m surprised these are `@computed` decorators are necessary, but I shall trust that they are.